packages/react-magma-dom/src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { css } from '@emotion/core';
+import * as CSS from 'csstype';
+
+export enum GridDisplay {
+  grid = 'grid', // default
+  inlineGrid = 'inline-grid',
+}
+
+export enum GridJustifyItems {
+  start = 'start',
+  end = 'end',
+  center = 'center',
+  stretch = 'stretch',
+}
+
+export enum GridJustifyContent {
+  start = 'start',
+  end = 'end',
+  center = 'center',
+  stretch = 'stretch',
+  spaceAround = 'space-around',
+  spaceBetween = 'space-between',
+  spaceEvenly = 'space-evenly',
+}
+
+export enum GridAlignItems {
+  start = 'start',
+  end = 'end',
+  center = 'center',
+  stretch = 'stretch',
+}
+
+export enum GridAlignContent {
+  start = 'start',
+  end = 'end',
+  center = 'center',
+  stretch = 'stretch',
+  spaceAround = 'space-around',
+  spaceBetween = 'space-between',
+  spaceEvenly = 'space-evenly',
+}
+
+export enum GridItemJustifySelf {
+  start = 'start',
+  end = 'end',
+  center = 'center',
+  stretch = 'stretch',
+}
+
+export enum GridItemAlignSelf {
+  start = 'start',
+  end = 'end',
+  center = 'center',
+  stretch = 'stretch',
+}
+
+export enum GridAutoFlow {
+  row = 'row',
+  column = 'column',
+  rowDense = 'row-dense',
+  columnDense = 'column-dense',
+}
+
+/**
+ * @children required
+ */
+export interface GridProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
since all the props like `gridDisplay` are in the `GridProps` should we drop all the prefaces?

`<Grid display={GridDisplay.inlineGrid}>` is a little friendlier than `<Grid gridDisplay={GridDisplay.inlineGrid}>` 

gets a little crazier with `<GridItem gridItemJustifySelf={GridItemJustifySelf.start}>` ¯\_(ツ)_/¯ 

I wouldn't remove them from the exported enums since those get exported from `react-magma-dom` but maybe from the property names ¯\_(ツ)_/¯ 

¯\_(ツ)_/¯ I dunno ¯\_(ツ)_/¯